pkgs/development/python-modules/dvc-data/default.nix

```diff
@@ -15,7 +15,7 @@
 
 buildPythonPackage rec {
   pname = "dvc-data";
-  version = "2.11.0";
+  version = "2.12.0";
```

Comment:
While reviewing, I noticed that version 2.12.1 was [just release](https://github.com/iterative/dvc-data/releases/tag/2.12.1).

Reply:
Thanks, updated.